qemu/tests/cfg/reject_qemu_img_info.cfg

```diff
@@ -0,0 +1,16 @@
+- reject_qemu_img_info:
+    only qcow2
+    virt_test_type = qemu
+    kill_vm = yes
+    start_vm = no
+    force_create_image = no
+    type = reject_qemu_img_info
+    variants:
+        - info_snapshot:
```

Comment:
It would be better to name it to "info_snapshot_chain"

Reply:
updated, thanks